client/my-sites/plan-features/index.jsx

```diff
@@ -90,6 +92,15 @@ export default connect( ( state, ownProps ) => {
 		rawPrice: getPlanRawPrice( state, planProductId /**, get from abtest **/ ),
 		planConstantObj: plansList[ ownProps.plan ],
 		billingTimeFrame: get( planObject, 'bill_period_label', '' ),
+		onUpgradeClick: ownProps.plan === PLAN_FREE ? noop : () => {
```

Comment:
Wouldn't be enough to put this into a fn in this file? Or is it a convention to put things like this in lib so they can be reused?